synapse/models/transport.py

```diff
@@ -5,19 +5,64 @@ def getModelDefs(self):
         modl = {
             'types': (
 
+                # TODO is transport:journey a thing?
+
+                ('transport:cargo', ('guid', {}), {
+                    'doc': 'Cargo being carried by a vehicle on trip.'}),
+
+                ('transport:point', ('str', {'lower': True, 'onespace': True}), {
+                    'doc': 'A departure/arrival point such as an airport gate or train platform.'}),
+
+                ('transport:trip', ('ndef', {'interface': 'transport:trip'}), {
+                    'doc': 'A trip such as a flight or train ride.'}),
+
+                ('transport:stop', ('guid', {}), {
+                    'interfaces': ('transport:schedule',),
+                    'doc': 'A stop made by a vehicle on a trip.'}),
+
+                ('transport:container', ('ndef', {'interface': 'transport:container'}), {
+                    'doc': 'A container capable of transporting cargo or personnel.'}),
+
+                ('transport:vehicle', ('ndef', {'interface': 'transport:vehicle'}), {
+                    'doc': 'A vehicle such as an aircraft or sea vessel.'}),
+
+                ('transport:occupant', ('guid', {}), {
+                    'doc': 'An occupant of a vehicle on a trip.'}),
+
+                ('transport:occupant:role:taxonomy', ('taxonomy', {}), {
+                    'interfaces': ('meta:taxonomy',),
+                    'doc': 'A taxonomy of transportation occupant roles.'}),
+
                 ('transport:direction', ('hugenum', {'modulo': 360}), {
                     'doc': 'A direction measured in degrees with 0.0 being true North.'}),
 
+                ('transport:land:vehicle:type:taxonomy', ('taxonomy', {}), {
```

Comment:
Seems weird to have a taxonomy just for land vehicles. Why not a taxonomy for vehicles in general that could be applied to land, sea, air vehicles?

Reply:
Each vehicle form is free to have ( or not ) a `:type` which is distinct from the other vehicle types. A unified type would always need to ( and probably would fail to ) start with `land` or `aircraft`. By making them distinct, we only populate the ones we need. We encounter the need for land vehicle types more frequently than plans/trains/vessels because land vehicles come in a wider variety. We may eventually add types to the others.

Reply:
Meh, on second thought, i've got some other minor edits. I'll add types for the others preemptively :) Thanks!